css/_uswds-theme-custom-styles.scss

```diff
@@ -2844,8 +2853,9 @@ $theme-table-column-gap: 4;
       @include u-text("uppercase", "ls-2", "normal");
     }
 
-    .site-text-intro {
-      @extend .site-text-intro;
+    .site-text-intro,
+    %site-text-intro {
+      @extend %site-text-intro;
```

Comment:
This placeholder selector should be defined before `.site-prose` and then extended here and in line `2893`.